### PR TITLE
Adapt to tlf #474

### PR DESCRIPTION
**Pull request by Felixmil**
[Recent changes in TLF](https://github.com/Open-Systems-Pharmacology/TLF-Library/pull/474) impact snapshots (slight different margin from `ggtext::element_textbox()`)
Plus name change of a method.

### CODE REVIEW

File: tests/testthat/_snaps/plot-population-time-profile/default-quantiles.svg
Comment (PavelBal) [inline tests/testthat/_snaps/plot-population-time-profile/default-quantiles.svg:1]: @Felixmil Why are legend captions wrapped here? What is the criteria for text wrap?

Comment (Felixmil) [reply tests/testthat/_snaps/plot-population-time-profile/default-quantiles.svg:1]: String length (50). Wrapped on spaces, and we have one here before the ().

Comment (PavelBal) [reply tests/testthat/_snaps/plot-population-time-profile/default-quantiles.svg:1]: This is arbitrary set, right? Or how did you come up with this value?

Comment (Felixmil) [reply tests/testthat/_snaps/plot-population-time-profile/default-quantiles.svg:1]: I came up with this number in TLF after running the package's tests and chose a value that looked right. This value can be customized.

Comment (Felixmil) [reply tests/testthat/_snaps/plot-population-time-profile/default-quantiles.svg:1]: This value can be customized using `tlf::setMaxCharacterWidth()`

``` r
devtools::load_all()
#> ℹ Loading ospsuite
#> Loading required package: rClr
#> 
#> Loading the dynamic library for Microsoft .NET runtime...
#> Loaded Common Language Runtime version 4.0.30319.42000
#> Warning: package 'testthat' was built under R version 4.3.1

# Load simulation
simFilePath <- system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
sim <- loadSimulation(simFilePath)

populationResults <- importResultsFromCSV(
  simulation = sim,
  filePaths = system.file("extdata", "SimResults_pop.csv", package = "ospsuite")
)

myDataComb <- DataCombined$new()
myDataComb$addSimulationResults(populationResults)

plotPopulationTimeProfile(myDataComb)
```



``` r

tlf::setMaxCharacterWidth(60)

plotPopulationTimeProfile(myDataComb)
```



``` r

tlf::setMaxCharacterWidth(999)

plotPopulationTimeProfile(myDataComb)
```



<sup>Created on 2023-08-10 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Review by PavelBal — APPROVED** [review]

**Pull request merged**